src/createRollupConfig.ts

```diff
@@ -198,11 +198,12 @@ export async function createRollupConfig(
       opts.env !== undefined &&
         replace({
           'process.env.NODE_ENV': JSON.stringify(opts.env),
+          preventAssignment: true,
         }),
       sourceMaps(),
       shouldMinify &&
         terser({
-          sourcemap: true,
+          // sourcemap: true,
```

Comment:
Because of this `https://github.com/TrySound/rollup-plugin-terser/releases/tag/v6.0.0` remove this line please

Reply:
thanks. done